Introduce code of conduct policy (#31)
cfpb/open-source-project-template: CODE_OF_CONDUCT.md

# CFPB Open Source Code of Conduct

## Introduction

The [Consumer Financial Protection Bureau](https://www.consumerfinance.gov) (CFPB) is committed to 
building a safe, welcoming, harassment-free culture for everyone. We do not merely want an 
environment that is free from hostility; we want one that is actively welcoming and inclusive. We 
want our team, our workplace culture, and our open source community to reflect and celebrate the 
diversity of the people we serve.

This Code of Conduct summarizes federal anti-harassment law and CFPB policy. 
Scope

We expect everyone on the CFPB team, and those contributing to our open source community, to exhibit 
these behaviors and abide by applicable federal laws and CFPB policies. In addition, we expect 
everyone within CFPB spaces to exhibit these behaviors and refrain from behavior prohibited by 

> anti-harassment laws and federal policies on harassment. These spaces include:

Reviewer: It would be nice to simplify/clarify this paragraph in a future revision to this COC.

It seems to be making several different distinctions that might be important:
- CFPB team vs. those contributing to our open source community vs. everyone within CFPB spaces
- "applicable federal laws and CFPB policies" vs. "anti-harassment laws and federal policies on harassment"
- everyone in CFPB spaces vs. [people somehow involved but _not_ in CFPB spaces?]
but it's not clear why these distinctions are made.

Author: I agree that this is a bit confusing

- CFPB’s physical offices,
- CFPB events and meetings,
- All of CFPB’s online forums and virtual collaboration tools, including code repositories

## What we strive for

> At CFPB, we strive to create a welcoming and inclusive culture that empowers people to best protect 

Reviewer: ```suggestion
At the CFPB, we strive to create a welcoming and inclusive culture that empowers people to best protect 
```

the financial interests of all consumers. That kind of atmosphere requires an open exchange of ideas 
balanced by thoughtful guidelines. Examples of behavior that contributes to a positive environment 
for our open source community include:

- Demonstrating empathy and kindness toward other people
- Being respectful of differing opinions, viewpoints, and experiences
- Giving and gracefully accepting constructive feedback
- Accepting responsibility and apologizing to those affected by our mistakes, and learning from the experience
- Focusing on what is best not just for us as individuals, but for the overall community and public

## Unacceptable behavior

To help understand the kinds of behaviors that are illegal or run counter to the culture we seek to 

> foster, we've listed below actions that violate federal law and CFPB policy. We've also included 

Reviewer: ```suggestion
foster, we've listed some actions below that violate federal law and CFPB policy. We've also included 
```

> foster, we've listed below actions that violate federal law and CFPB policy. We've also included 

Reviewer: I'm a little uncomfortable with the inclusion of "illegal" and "violate federal law" in this sentence, without making it clear below which of these things might be actual violations of law. The lists below only refer to our policy.

Author: This is in line with the wording of the [18F/TTS code of conduct](https://18f.gsa.gov/code-of-conduct/), but noted for a future iteration.

- Uniformed status
- Membership in a labor organization or union activities
- Prior equal employment opportunity (EEO) or whistleblower activity, or
- Any other factor unrelated to your merit

The policy also forbids harassing conduct, which includes unwelcome conduct based on any (or a combination of) protected traits or characteristics. Such conduct may take the form of any of the following: 

- Offensive jokes, comments, objects, or pictures 
- Questions about a person’s identity (e.g., disability status, gender identity, sexual orientation, national origin, etc.) 
- Undue attention 
- Ridicule or mockery 
- Insults or put-downs 
- Touching/physical contact 
- Slurs or epithets 
- Threats or other forms of intimidation 
- Physical or sexual assault

## Reporting violations

If you are a CFPB employee, former CFPB employee, or job applicant to CFPB and believe you have been 
discriminated against or harassed on the basis of race, color, religion, sex (including pregnancy, 
sexual orientation, transgender status, gender identity or expression, gender non-conformity, or sex 
stereotyping of any kind), national origin, disability, age (40 years or older), genetic information, 

> parental status or retaliated against for prior EEO activity, contact the CFPB’s EEO Office.

Reviewer: ```suggestion
or parental status, or retaliated against for prior EEO activity, contact the CFPB’s Office of Civil Rights.
```

Reviewer: I think these other suggestions got lost when you made the change to spell out EEO.

Author: I'm going to add the comma, but keeping the "or" where it is since that is copied directly from EEO language used elsewhere

CFPB_EEO@consumerfinance.gov

(202) 435-9EEO
(855) 233-0362
TTY: (202) 435-9742

Office of Civil Rights
Consumer Financial Protection Bureau
1700 G Street, NW
Washington, D.C. 20552

For help filing a complaint about discrimination on the basis of marital status, political 
affiliation, or any other non-merit factor, or for claims of retaliation for whistleblower activity, 

> contact the Office of Special Counsel  or the Merit Systems Protection Board .

Reviewer: ```suggestion
contact the Office of Special Counsel or the Merit Systems Protection Board.
```

For help filing a complaint about discrimination on the basis of uniformed status, you may contact 

> the Veterans’ Employment and Training Service (VETS)  at the Department of Labor, the Merit Systems 
> Protection Board , or the Office of Special Counsel , depending on the circumstances.

Reviewer: ```suggestion
the Veterans’ Employment and Training Service (VETS) at the Department of Labor, the Merit Systems 
Protection Board, or the Office of Special Counsel, depending on the circumstances.
```

For help filing a complaint about discrimination on the basis of membership in a labor organization, 

> you may contact the Federal Labor Relations Authority  or your union (if applicable).

Reviewer: ```suggestion
you may contact the Federal Labor Relations Authority or your union (if applicable).
```

### Equal employment opportunity policy

> For more information about CFPB’s equal employment opportunity (EEO) policyies and procedures visit https://www.consumerfinance.gov/office-civil-rights/eeo-policy-and-reports/ 

Reviewer: ```suggestion
For more information about the CFPB’s equal employment opportunity (EEO) policies and procedures, visit https://www.consumerfinance.gov/office-civil-rights/eeo-policy-and-reports/.
```

Reviewer: Missed the comma and period suggested here.

## Credits

> CFPB is greatly appreciative of the multiple sources that we drew from to build this Code of Conduct, including:

Reviewer: ```suggestion
The CFPB is greatly appreciative of the multiple sources that we drew from to build this Code of Conduct, including:
```
